scrapy/spiders/__init__.py

```diff
@@ -87,7 +87,7 @@ def make_requests_from_url(self, url):
         return Request(url, dont_filter=True)
 
     def parse(self, response):
-        raise NotImplementedError
+        raise NotImplementedError('Spider.parse callback is not defined')
```

Comment:
sorry for last call review, What do you think about formatting `Spider.parse` as `'{self.__class__.__name__}.parse ...` so it shows correct class name instead of generic _Spider_?